components/operators/openshift-gitops/instance/base/gitops-rbac.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ClusterRoleBinding
```

Comment:
This cluster role already exists as a component here:

https://github.com/redhat-composer-ai/cluster-gitops/tree/main/components/operators/openshift-gitops/instance/components/application-controller-cluster-admin

The cluster scoped instance already applies that component so we just need to add it to the composer instance here:

https://github.com/redhat-composer-ai/cluster-gitops/blob/main/tenants/composer-ai/argocd/base/kustomization.yaml

Reply:
Removed for now